sql/amo_prod/amo_stats_installs_v2/init.sql

```diff
@@ -0,0 +1,50 @@
+CREATE OR REPLACE TABLE
+  `moz-fx-data-shared-prod.amo_prod.amo_stats_installs_v2`
+PARTITION BY
+  submission_date
+CLUSTER BY
+  addon_id
+AS
+WITH installs AS (
+    --
+  SELECT
+    client_id,
+    submission_date,
+    udf.get_key(event_map_values, 'addon_id') AS addon_id,
+    -- `step` is needed to only get one record for 1 download/install.
+    udf.get_key(event_map_values, 'step') AS extra_step,
+    -- We need to filter out installs not initiated from AMO (or DISCO).
+    udf.get_key(event_map_values, 'source') AS extra_source,
+    -- We retain `method` here to show how we could retrieve
+    -- attribution data instead (this isn't collected yet).
+    -- For AMO, we do NOT need this field. We'll have something
+    -- like `extra_attribution` or `extra_utm_params` probably.
+    udf.get_key(event_map_values, 'method') AS extra_method,
```

Comment:
In this case, `method` contains things like `link` or `amWebAPI`, etc. We could think of `method` here like `utm_source` in the future, so `extra_method` should be later "converted" to a column that contains:

```
{
  'link': 1234,
  'amWebAPI': 4567
}
```

(similar to `dau_by_app_version` in `amo_stats_dau` for example)

Reply:
Do you want me to go ahead and create a `downloads_by_method` column? Or should we just remove this completely?

Reply:
I think we can remove it completely for now.